lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1082,6 +1163,9 @@ bool ClangImporter::importBridgingHeader(StringRef header, ModuleDecl *adapter,
     Impl.handleDeferredImports();
     return false;
   }
+
+  Impl.setSinglePCHImport(None);
```

Comment:
I think you need to put this in importHeader, not importBridgingHeader. (That's what Serialization calls.)